src/pages/teams/[teamId]/dashboard/residents/[residentId]/document/[documentSubmissionId].tsx

```diff
@@ -180,6 +182,10 @@ const DocumentDetailPage: NextPage<WithUser<DocumentDetailPageProps>> = ({
         </title>
       </Head>
 
+      {documentSubmission.validUntil < DateTime.local() ? (
+        <PageWarning />
+      ) : null}
```

Comment:
You could update this to `{documentSubmission.validUntil < DateTime.local() && (` then you shouldn't need `null`

Reply:
And would be good to cover this case with a Cypress test

Reply:
👍